stdlib/2.7/__builtin__.pyi

```diff
@@ -814,7 +814,11 @@ class OSError(EnvironmentError): ...
 class WindowsError(OSError): ...
 class OverflowError(ArithmeticError): ...
 class ReferenceError(StandardError): ...
-class SyntaxError(StandardError): ...
+class SyntaxError(StandardError):
+    msg = ...  # type: str
+    lineno = ...  # type: Union[int, long]
+    offset = ...  # type: Union[int, long]
```

Comment:
Can you make these plain int? We don't really care about the difference between int and long, and mypy treats them the same. There are (almost) no other stubs mentioning the long type.